DEBUGGING.md

```diff
@@ -0,0 +1,21 @@
+# General debugging tips
+This file answers some commmon questions that arise when you are writing a fuzzer using LibAFL.
+
+## Q. My fuzzer crashed but the stack trace is useless.
+You can enable `errors_backtrace` feature of `libafl` crate. With this the stacktrace is meaningful.
+
+## Q. I started the fuzzer but the corpus count is 0.
+Unless the initial corpus is loaded with the "load_initial_inputs_forced" function, we only store the interesting inputs, which is the inputs that triggered the feedback. So this usually means that your input was not interesting or your target was simply not propoerly implemented.

```

Comment:
```suggestion
Unless the initial corpus is loaded with the "load_initial_inputs_forced" function, we only store the interesting inputs, which is the inputs that triggered the feedback. So this usually means that your input was not interesting or your target was simply not properly implemented.
```

Reply:
A solution/debugging tip for either scenario would go a long way:

1. How to confirm that my input was not interesting?
2. How to check if target is not properly implemented. This one can be tricky but common pit falls could be mentioned at least.